packages/client/utils/AtlassianManager.ts

```diff
@@ -335,7 +335,7 @@ export default abstract class AtlassianManager {
       return new Error(json.errorMessages[0])
     }
     if ('errors' in json) {
-      const errorFieldName = Object.keys(json.errors)[0]
+      const errorFieldName = Object.keys(json.errors)[0] as string
```

Comment:
If this line is correct in the first place, please fix type in `JiraFieldError` instead